tests/test_coin_combination.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.coin_combination import get_coin_combination
 
-# write your tests here
+
+@pytest.mark.parametrize(
+    'coins,result',
+    [
+        [1, [1, 0, 0, 0]],
```

Comment:
Please, use `pytest.params` to provide a description for each test with `id` field